model/common_structures.smithy

```diff
@@ -59,3 +59,28 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+list UserList {
+    member: User
+}
+
+structure User {
+    hash: String,
+    reserved: Boolean,
+    hidden: Boolean,
+    backend_roles: BackendRolesList,
+    attributes: AttributeMap,
+    description: String,
+    opendistro_security_roles: OpendistroSecurityRolesList,
+    static: Boolean
+}
+
+map AttributeMap {
+    key: String,
+    value: String
+}
+
+structure GeneralResponse{
```

Comment:
`GenericResponse`

Reply:
epic fail XD
Sorry! fixed that, already 1:24 am